src/gluonts/ev/metrics.py

```diff
@@ -565,7 +565,7 @@ def mean(
             [np.abs(coverages[f"coverage[{q}]"] - q) for q in quantile_levels],
```

Comment:
I think I'm missing some context on the implementation of these methods & numpy masked array behavior to catch the potential edge cases. 

I would recommend just extensively testing the existing metric implementations, e.g., by ensuring matching values w.r.t. the old Evaluator, including cases like predictions / past data / future data are all zeros / all constant, etc.